dev/tools/deprecate_file

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
Maybe the file can be named `deprecate_file.sh` like others in the same directory?

Reply:
done (notice the tabs changed to spaces for indentation, this is done by git-fix-ws-tree, although I guess tabs in sh scripts are ok?).

Reply:
I hope so

Reply:
Absence of tab checking is controlled by .gitattributes which is extension based, so an extension-less file avoids it.